pkg/sql/conn_executor.go

```diff
@@ -1645,7 +1645,8 @@ func (ex *connExecutor) execCopyIn(
 		ex.initPlanner(ctx, p)
 		ex.resetPlanner(ctx, p, txn, stmtTS, 0 /* numAnnotations */)
 	}
-	if cmd.Stmt.Table.TableName == fileUploadTable {
+	table := cmd.Stmt.Table
```

Comment:
merge this line into the if:
`if table := cmd.Stmt.Table; table.Table() == fileUploadTable ...`